.github/workflows/main.yml

```diff
@@ -23,6 +23,11 @@ jobs:
           go-version: '1.x'
       - name: Install Dependencies
         run: sudo apt-get install make curl jq
+      - name: Install pack
```

Comment:
Should we instead use https://github.com/buildpacks/github-actions#setup-pack-cli-action ?